src/dmtcp_launch.cpp

```diff
@@ -535,7 +535,7 @@ main(int argc, const char **argv)
   }
 
   // Test and set env var for FSGSBASE. The env var is used by MANA.
-  testFsGsBase();
+  //testFsGsBase();
```

Comment:
This commit is no longer necessary.  In the latest 'main' branch, `testFsGsBase()` is called only for `x86_64`.  Please remove this commit.

Even worse, x86_64 wants this code.  We shouldn't disturb x86_64.